tortoise/verifying_tortoise_test.go

```diff
@@ -3063,3 +3108,42 @@ func TestVoteBlockFilterForHealing(t *testing.T) {
 	assert.True(t, filter(goodBlock.ID()))
 	assert.True(t, filter(badBlock.ID()))
 }
+
+func TestSelfHealingCountBadBeaconBlocks(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+
+	// increase layer size reduce test flakiness
+	s := sim.New(sim.WithLayerSize(defaultTestLayerSize * 10))
```

Comment:
generator should be in charge of the state. this commit makes it depend on additional mocking, if beacons need to be supported by sim module then sim module should store it somewhere, preferably in the same database that will be used by tortoise beacon, so that benchmarking is simplified

Reply:
removed.